documentation/src/main/asciidoc/stories/assembly_partition_handling.adoc

```diff
@@ -1,8 +1,23 @@
 [id='partition-handling']
 :context: partition-handling
-= Setting up partition handling
+= Configuring {brandname} to handle network partitions
+
+{brandname} clusters can split into network partitions in which subsets of nodes become isolated from each other.
+This condition results in loss of availability or consistency for clustered caches.
+To handle network partitions, {brandname} automatically detects crashed nodes and resolves conflicts to merge caches back together.
 
 include::{topics}/con_partition_handling.adoc[leveloffset=+1]
+include::{topics}/con_partition_handling_recovery_example.adoc[leveloffset=+2]
+include::{topics}/con_partition_handling_data_consistency.adoc[leveloffset=+2]
+include::{topics}/con_partition_handling_garbage_collection.adoc[leveloffset=+2]
```

Comment:
I think this should be the last thing to include, as all other sections relate to explaining and overcoming the problem. Whereas this feels like a warning about how to avoid the issue in the first place.